packages/react-components/style/output-components-style.css

```diff
@@ -93,27 +97,80 @@ canvas {
     align-self: center;
 }
 
-.table-tree>tbody>tr:nth-child(1)>th {
-    background-color: var(--theia-editor-background);
-    position: sticky;
-    top: 0;
+.table-tree {
```

Comment:
I think it looks good to highlight the row when hovering on a row... the events table, file explorer, open trace view etc. do that. I only manged to highlight a cell by adding the lines below. Maybe you can figure it out.

.table-tree td:hover {
    background-color: var(--theia-list-hoverBackground) !important;
}

Reply:
Here is a solution to highlight the whole line. I suggest adding it below line 138.
```
.table-tree td:hover, .table-tree td:hover ~ td {
    background-color: var(--theia-list-hoverBackground);
}
```

Reply:
I think you need the ` !important` at the end.

Reply:
Not needed:
<img width="998" alt="Screenshot 2021-12-21 at 15 35 53" src="https://user-images.githubusercontent.com/88502808/146994371-c39ff3bf-69a2-4d53-84ec-b5c45a1a6af0.png">

Reply:
I just saw it for all other `var(...)` references in that css file and thought it would be needed.

Reply:
> Here is a solution to highlight the whole line. I suggest adding it below line 138.
> 
> ```
> .table-tree td:hover, .table-tree td:hover ~ td {
>     background-color: var(--theia-list-hoverBackground);
> }
> ```

This is some kind of sibling notation right? Good to know!

I had missed your comment and found that this also works:

.table-tree tr:hover td {

Reply:
Yes Patrick, it's using CSS siblings. But your solution works, too.
And we should only use `!important` when really necessary, for that it breaks CSS inheritance. In this case, it works fine without it.